stable/admin/templates/NOTES.txt

```diff
@@ -6,4 +6,4 @@ NuoDB can be accessed via port 48004 on the following DNS name from within your
 
 To display your NuoDB cluster state run:
 
-  $ kubectl exec admin-{{ .Values.admin.domain }}-0 -- nuocmd show domain
+  $ kubectl exec admin-{{ .Values.admin.domain }}-{{ .Values.cloud.cluster.name }}-0 -- nuocmd show domain
```

Comment:
To be more accurate I suggest that we use the following template:
```
$ kubectl exec {{ include "admin.fullname" . }}-0 -- nuocmd show domain
```

Reply:
+1